### ISSUE TITLE
How to apply patches ?

### ISSUE THREAD

**Issue by EndlessPeak** [leonmaxx/asus_fanmode#3]
I am a new bud to Linux.
Since I am using Manjaro Linux 5.4-35 on the ASUS laptop,I am terribly shame to ask my question,which sounds quite silly:
How to apply the patches you mentioned in "readme.md" from "https://patchwork.kernel.org" to my linux kernel?
Could you please give some docs link,tips or something just make me understand how to deal with it?Thank you so much.

**Comment by leonmaxx** [comment]
Hey,
What model of your laptop? Possibly you don't need to update anything.
  
On Manjaro you can update kernel to version 5.6 with the help of `mhwd-kernel`, open terminal and type:  

```
sudo mhwd-kernel -i linux56
```

**Comment by EndlessPeak** [comment]
I have installed the linux56 kernel by mhwd successfully,but the service still doesn't work well.
My laptop model is ASUS TUF Gaming FX80.CPU core i5-8300H,GPU GTX1050 Ti,RAM 8GB.
Operating system is Manjaro Linux 5.6.8-1.
Here is the error log.
![1](https://user-images.githubusercontent.com/53530553/81054483-82c26400-8ef9-11ea-83ca-d4ba534ae2ae.png)
Last time the error is switch fan mode from "normal" to "silent",and report error either.
I tried to edit the file "/sys/devices/platform/asus-nb-wmi/fan_boost_mode", after I edit it from "2"(normal mode) to "1"(overboost mode),the problem appeared:though the fan starts to speed up and works well,my laptop's keyboard lose control.I cannot do anything but reboot compulsively.
After reboot,I tried fn+F5,it doesn't work.
Is there anything need to config or fix?
Thank you again for your kindness!
